ACE/ace/CDR_Base.h

```diff
@@ -335,10 +335,11 @@ class ACE_Export ACE_CDR
          {
            MAX_DIGITS = 31,
            MAX_STRING_SIZE = 4 + MAX_DIGITS, // includes -, 0, ., terminator
-           POSITIVE = 0xc,
-           NEGATIVE = 0xd
          };
 
+         static const Octet POSITIVE = 0xc;
```

Comment:
Why only mode these out of the enum? constexpr instead of const?

Reply:
They're used differently from the other constants because they are part of expressions that are assigned to values in the byte array representation of `Fixed`.  Having them start with type `Octet` is easier than needing to convert each time they're used.

The compiler treats `static const` members of classes which have integer types and are assigned values in the class definition the same as constexpr, and then the code can exactly match across the different ACE versions.